data/appstream/meson.build

```diff
@@ -0,0 +1,8 @@
+i18n.merge_file(
+  'gpaste-appdata',
+  input: 'org.gnome.GPaste.Ui.appdata.xml.in',
+  po_dir: '../../po',
```

Comment:
Is there a way not to hardcode the `../..`? Something like `root_directory` or such?

Reply:
We could use `source_root`, but that would break building GPaste as a subproject.